include/zigbee/zigbee_error_handler.rst

```diff
@@ -0,0 +1,31 @@
+.. _lib_zigbee_error_handler:
+
+Zigbee error handler
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Zigbee error handler library provides error codes for ZBOSS functions that return codes other than ``RET_OK``.
```

Comment:
No, it does not. The error codes are provided as a part of ZBOSS API.
This section has to be refactored.